### DISCUSSION TITLE
Invalid config after upgrading to 7.0.4

### DISCUSSION THREAD

**Discussion by prnewman** [fuatakgun/eufy_security#909] [General]
Got the following error after upgrading to 7.0.4 and restarting:
![image](https://github.com/fuatakgun/eufy_security/assets/857199/1e1d6c6a-a0aa-4b8a-825a-54671e770a22)

Enabled logging but don't see any errors relating to eufy_security.

Tried deleting from Integrations to reinstall but deleting didn't remove Eufy Security integration. 
![image](https://github.com/fuatakgun/eufy_security/assets/857199/aaf2d266-2533-4271-8e2d-70922a112fb6)

Wondering if I should roll back to 6.3.3. 

**Environment:**
Version: core-2023.5.4
Installation Type: Home Assistant OS
eufy-security-ws (1.6.3)

**Comment by fuatakgun** [comment]
How can we help with little information? You need to share the logs.

**Comment by prnewman** [comment]
I added this line to configuration.yaml and restarted, but I'm not seeing debug messages for eufy_security in the log files:

`custom_components.eufy_security: debug`

**Comment by prnewman** [comment]
eufy-security-ws log:

```
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
2023-08-28 21:38:55.095  INFO  Eufy Security server listening on host 0.0.0.0, port 3000 
2023-08-28 21:39:00.165  INFO  Connected to station T8010P1320220957 on host 192.168.86.31 and port 22700 
2023-08-28 21:39:00.445  INFO  MQTT connection successfully established 
2023-08-28 21:39:00.494  INFO  Successfully registered to MQTT notifications for lock T8520Q1021170B36 
2023-08-28 21:39:12.563  INFO  Push notification connection successfully established 
2023-08-28 21:39:20.583  WARN Station T8520Q1021170B36 - Tried all hosts, no connection could be established 
2023-08-28 21:39:20.586  INFO  Timeout connecting to station T8520Q1021170B36
```

**Comment by prnewman** [comment]
I noticed that RTSP Simple Server Add-on failed to start:

Failed to start add-on
Port '8554' is already in use by something else on the host.

**Reply by fuatakgun** [reply]
You can change the port to something else.

You probably have rtsptowebrtc or go2rtc

**Reply by prnewman** [reply]
Changed the port and it starts now. Rebooted but eufy_security invalid config error persists. 

At this point I'm stumped. Any advice on completely removing integration to reinstall when the Delete button doesn't work?

Thanks in advance for your help.

**Comment by prnewman** [comment]
I'm guessing I need to manually remove the eufy_security integration from `core.config_entries` in `.storage` and restart.

![image](https://github.com/fuatakgun/eufy_security/assets/857199/8e86e6be-fbcb-46a0-90fd-68aaba81173b)

Anything about this config seem unusual? I see that `use_rtsp_server_addon` is set to false, and of course the port is now wrong.

```
      {
        "entry_id": "cc18bf7940c7aa16e491388619a66837",
        "version": 1,
        "domain": "eufy_security",
        "title": "localhost",
        "data": {
          "host": "localhost",
          "port": 3000
        },
        "options": {
          "sync_interval": 600,
          "use_rtsp_server_addon": false,
          "rtsp_server_address": "localhost",
          "rtsp_server_port": 8554,
          "ffmpeg_analyze_duration": 1.2,
          "auto_start_stream": false,
          "fix_binary_sensor_state": true,
          "map_extra_alarm_modes": false,
          "name_for_custom1": "Custom 1",
          "name_for_custom2": "Custom 2",
          "name_for_custom3": "Custom 3"
        },
        "pref_disable_new_entities": false,
        "pref_disable_polling": false,
        "source": "user",
        "unique_id": null,
        "disabled_by": null
      },

```

**Comment by prnewman** [comment]
Restored to v6.3.2 from a full backup and Eufy Security is working again. Never identified root cause.
